### PR TITLE
Factory:ARM: raspberrypi3 JeOS renamed to raspberrypi

### PR DESCRIPTION
**Pull request by ggardet**

### CODE REVIEW

File: t/obs/openSUSE:Factory:ARM:ToTest/jeos-aarch64/print_rsync_iso.before [hunk 1]
```diff
@@ -1,7 +1,7 @@
...
-rsync --timeout=3600 -tlp4 --specials obspublish::openqa/openSUSE:Factory:ARM:ToTest/appliances/aarch64/JeOS:JeOS-raspberrypi3.aarch64/*openSUSE-Tumbleweed-ARM-JeOS-raspberrypi3.aarch64-2020.01.08-Snapshot20200115.raw.xz.sha256 /var/lib/openqa/factory/other/openSUSE-Tumbleweed-ARM-JeOS-raspberrypi3.aarch64-2020.01.08-Snapshot20200115.raw.xz.sha256
+rsync --timeout=3600 -tlp4 --specials obspublish::openqa/openSUSE:Factory:ARM:ToTest/appliances/aarch64/JeOS:JeOS-raspberrypi.aarch64/*openSUSE-Tumbleweed-ARM-JeOS-raspberrypi3.aarch64-2020.01.08-Snapshot20200115.raw.xz /var/lib/openqa/factory/hdd/openSUSE-Tumbleweed-ARM-JeOS-raspberrypi3.aarch64-2020.01.08-Snapshot20200115.raw.xz
+rsync --timeout=3600 -tlp4 --specials obspublish::openqa/openSUSE:Factory:ARM:ToTest/appliances/aarch64/JeOS:JeOS-raspberrypi.aarch64/*openSUSE-Tumbleweed-ARM-JeOS-raspberrypi3.aarch64-2020.01.08-Snapshot20200115.raw.xz.sha256 /var/lib/openqa/factory/other/openSUSE-Tumbleweed-ARM-JeOS-raspberrypi3.aarch64-2020.01.08-Snapshot20200115.raw.xz.sha256
```
Comment (Vogtinator) [inline t/obs/openSUSE:Factory:ARM:ToTest/jeos-aarch64/print_rsync_iso.before:4]: Still says `raspberrypi3` later on. The openQA trigger should also change?

Comment (ggardet) [reply t/obs/openSUSE:Factory:ARM:ToTest/jeos-aarch64/print_rsync_iso.before:4]: I fail to find which file must be updated as I think `t/obs/openSUSE:Factory:ARM:ToTest/jeos-aarch64/print_rsync_iso.before` is generated.

Comment (andrii-suse) [reply t/obs/openSUSE:Factory:ARM:ToTest/jeos-aarch64/print_rsync_iso.before:4]: Short: need to change raspberrypi3 as well in https://github.com/os-autoinst/openqa-trigger-from-obs/blob/master/t/obs/openSUSE:Factory:ARM:ToTest/jeos-aarch64/files_iso.lst#L2

Long explanation: 
So the sync is performed in 3 steps:
1. read list of files from obs
2. generate bash commands for rsync and openqa calls, based on filenames fetched in step 1.
3. execute the generated commands

The tests cover only step 2, which is the trickiest. So list of files (step 1) must be provided manually inthe test.

Comment (ggardet) [reply t/obs/openSUSE:Factory:ARM:ToTest/jeos-aarch64/print_rsync_iso.before]: Ok, so my latest push should fix it. Thanks.

**Review by andrii-suse — APPROVED** [review]

**Pull request merged**